README.md

```diff
@@ -9,7 +9,14 @@ A Rust wrapper for libevdev
 ```toml
 # Cargo.toml
 [dependencies]
-evdev-rs = "0.3.1"
+evdev-rs = "0.4.0"
+```
+
+to enable serialization support, enable the feature "serde"
```

Comment:
We should also add a note about serde feature in the main documentation page which is generated from the first comment block in src/lib.rs